examples/colab/wav2vec2_saved_model_finetuning.ipynb

```diff
@@ -0,0 +1,2441 @@
+{
```

Comment:
<p>build -&gt; built</p><p> </p><p>when model.<strong>call</strong> or model.build are called, so ...</p><p> </p><p>for checking -&gt; to check</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/hub/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>